news/2025/2025-02-27-introducing-teams.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: Introducing Teams
+date: 2025-02-27 12:00:00 +0000
+---
+
+These are not "clans", they're not "subdivisions", nor are they "guilds" — they are **teams**! And they're here to stay.
+
+![](/wiki/shared/news/2025-02-27-introducing-teams/banner.jpg)
+
+It's time for a [15-year-old feature request](https://osu.ppy.sh/community/forums/topics/29780) to see reality. This feature which was once requested by [[Sean]](https://osu.ppy.sh/users/202721) exactly 14 years, 9 months and 10 days ago, ended up amassing a total priority of **+11,744** over the years. An idea which was widely supported by the community up to this day has finally come into fruition!
```

Comment:
would be +8 days if we counted from today

```suggestion
It's time for a [15-year-old feature request](https://osu.ppy.sh/community/forums/topics/29780) to see reality. This feature which was once requested by [[Sean]](https://osu.ppy.sh/users/202721) 14 years, 9 months and 10 days before it would be implemented, ended up amassing a total star priority of **+11,744** over the years. An idea which was widely supported by the community up to this day has finally come into fruition!
```